src/hardforks/petersburg.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
These changes seem correct according to my understanding the the meta EIP https://github.com/ethereum/EIPs/pull/1716/files and the core devs call on the issue https://github.com/ethereum/pm/blob/master/All%20Core%20Devs%20Meetings/Meeting%2053.md